src/NHSD.BuyingCatalogue.Ordering.Api/Models/OrderModel.cs

```diff
@@ -1,19 +1,19 @@
 ﻿using System;
+using System.Collections.Generic;
 
 namespace NHSD.BuyingCatalogue.Ordering.Api.Models
 {
     public sealed class OrderModel
     {
-        public string OrderId { get; set; }
-
         public string Description { get; set; }
-
-        public string Status { get; set; }
-
-        public DateTime DateCreated { get; set; }
-
-        public DateTime LastUpdated { get; set; }
-
-        public string LastUpdatedBy { get; set; }
+        public OrderingPartyModel OrderParty { get; set; }
+        public SupplierModel Supplier { get; set; }
+        public DateTime? CommencementDate { get; set; }
+        public IEnumerable<OrderItemModel> OrderItems { get; set; }
+        public IEnumerable<ServiceRecipientModel> UnusedServiceRecipients { get; set; }
```

Comment:
Due to a late change to the JSON contract should this now be `ServiceRecipients` rather than `UnusedServiceRecipients`?